python/pyspark/ml/functions.py

```diff
@@ -486,6 +486,7 @@ def predict(inputs: np.ndarray) -> np.ndarray:
     data = np.arange(0, 1000, dtype=np.float64).reshape(-1, 4)
```

Comment:
Can we follow https://numpydoc.readthedocs.io/en/latest/format.html and add the examples as a proper doctest?